webpack/assets/javascripts/react_app/components/Layout/components/TaxonomySwitcher/TaxonomyDropdown.js

```diff
@@ -81,7 +94,14 @@ const TaxonomyDropdown = ({ taxonomyType, currentTaxonomy, taxonomies }) => {
   return (
     <ContextSelector
       id={id}
-      toggleText={currentTaxonomy || anyTaxonomyText}
+      toggleText={
+        currentTaxonomy || (
+          <div>
+            {anyIcon}
+            {anyTaxonomyText}
+          </div>
```

Comment:
we can use `React.Fragment` instead of a `div` which currently adds additional DOM element:
```suggestion
          <>
            {anyIcon}
            {anyTaxonomyText}
          </>
```